### DISCUSSION TITLE
Closed toggle by default

### DISCUSSION THREAD

**Discussion by yuxxeun** [shuding/nextra#1584] [Q&A] [answered]
How to set the dropdown menu to closed by default? 

![gambar](https://user-images.githubusercontent.com/50067865/222353564-bf8de068-156e-408f-9c4d-5737d729d367.png)

**Comment by fuma-nama** [comment, marked as answer]
You can customise this behaviour by changing the `sidebar.defaultMenuCollapseLevel` in the theme configuration file

Take a look to [here](https://nextra.site/docs/docs-theme/theme-configuration#sidebar) for further details
